src/main/java/edu/ucla/library/prl/harvester/InvalidInstitutionJsonException.java

```diff
@@ -0,0 +1,54 @@
+package edu.ucla.library.prl.harvester;
+
+import info.freelibrary.util.StringUtils;
+
+/**
+ * Represents an error in the JSON representation of an {@link Institution}.
+ */
+public class InvalidInstitutionJsonException extends IllegalArgumentException {
+
+    /**
+     * The <code>serialVersionUID</code> for this class.
+     */
+    private static final long serialVersionUID = 1269493839347809483L;
+
+    /**
+     * The message unique to this exception.
+     */
+    private static final String MESSAGE = "The supplied JsonObject does not represent a valid Institution";
```

Comment:
This text should come from the resources file. There is an I18nRuntimeException class that can be extended that makes this easier (https://javadoc.io/doc/info.freelibrary/freelib-utils/latest/info/freelibrary/util/I18nRuntimeException.html). Also, if you really want to maintain that connection to the illegal argument idea, there is an IllegalArgumentI18nException (https://javadoc.io/doc/info.freelibrary/freelib-utils/latest/info/freelibrary/util/IllegalArgumentI18nException.html) that can be extended instead.

Reply:
Agreed on I18nRuntimeException